drivers/SmartThings/matter-window-covering/fingerprints.yml

```
@@ -26,6 +26,11 @@ matterManufacturer:
     vendorId: 0x139C
     productId: 0xFF15
     deviceProfileName: window-covering
+  - id: 5020/64050
```

To keep with the format of other `Zemismart` fingerprints
```suggestion
  - id: "5020/64050"
```